packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [34.0.0]
+
+### Changed
+
+- Enable `resetMocks` Jest configuration option ([#4417](https://github.com/MetaMask/core/pull/4417))
+- **BREAKING** Update Assets Controllers ([#4219](https://github.com/MetaMask/core/pull/4219))
```

Comment:
Instead of grouping together the changes in this PR, what are your thoughts on listing them separately? So instead of 

```
- **BREAKING** Update Assets Controllers ([#4219](https://github.com/MetaMask/core/pull/4219))
  - `TokenBalancesController` messenger must allow the action `AccountsController:getSelectedAccount` and remove `PreferencesController:getState`.
  - `TokenDetectionController` messenger must allow the action `AccountsController:getAccount`.
  - `TokenDetectionController` messenger must allow the event `AccountsController:selectedEvmAccountChange` and remove `AccountsController:selectedAccountChange`.
  ...
```

you have

```
- **BREAKING:** `TokenBalancesController` messenger must allow the action `AccountsController:getSelectedAccount` and remove `PreferencesController:getState` ([#4219](https://github.com/MetaMask/core/pull/4219))
- **BREAKING:** `TokenDetectionController` messenger must allow the action `AccountsController:getAccount` ([#4219](https://github.com/MetaMask/core/pull/4219))
- **BREAKING:** `TokenDetectionController` messenger must allow the event `AccountsController:selectedEvmAccountChange` and remove `AccountsController:selectedAccountChange` ([#4219](https://github.com/MetaMask/core/pull/4219))
...
```

This allows you to place these changes in separate categories if need be (for instance, anything that talks about removing can be listed in Removed rather than Changed).